src/hotspot/share/runtime/thread.cpp

```diff
@@ -695,8 +607,6 @@ bool JavaThread::is_ext_suspend_completed(bool called_by_wait, int delay,
       // make a single unnecessary pass through the logic below. This
       // doesn't hurt anything since we still do the trans retry.
 
-      *bits |= 0x00004000;
```

Comment:
In the comment above:
```
      // ... If we were
      // called by wait_for_ext_suspend_completion(), then it
      // will be doing the retries so we don't have to.
```
`wait_for_ext_suspend_completion()` has been deleted so the
comment needs work.